src/wazuh_db/wdb_parser.c

```diff
@@ -5558,6 +5566,38 @@ int wdb_parse_global_sync_agent_info_set(wdb_t * wdb, char * input, char * outpu
     return OS_SUCCESS;
 }
 
+int wdb_parse_get_groups_integrity(wdb_t* wdb, char* input, char* output) {
+    cJSON *response = NULL;
+    char *out = NULL;
+    char *hash = input;
+    int ret = OS_SUCCESS;
+
+    response = cJSON_CreateArray();
+    int groups_synced = wdb_global_groups_synced(wdb);
+
+    if (groups_synced == 1) {
+        cJSON_AddItemToArray(response, cJSON_CreateString("syncreq"));
+    } else if (groups_synced == 0) {
+        if (wdb_get_global_group_hash(wdb, hash)){
```

Comment:
This was a bad explanation of the dummy **wdb_get_global_group_hash()**. With this name, is expected that **wdb_get_global_group_hash** returns a hash. The dummy method should return a hash and we must verify it in a different method.

Reply:
Modified!